bibliopixel/drivers/__init__.py

```diff
@@ -0,0 +1 @@
+from . driver_base import ChannelOrder
```

Comment:
ChannelOrder had to be imported as bibliopixel.drivers.driver_base.ChannelOrder when using it in a script which is no good.